### ISSUE TITLE
NVDA cannot report parameter prompts in Visual Studio

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#12902]
This Issue comes from discussions #12716

I use C# to program, and I want to know how to read the parameter information of each method. Visual Studio has a shortcut key, Ctrl+Shift+Space or CtrL +k, p which is useless for me. When I use the above shortcut When you press the key, NVDA will only read "Ready", but using the up and down arrows does not read the given parameter information. Is this a bug in NVDA? Microsoft has related tutorials. The following is a link. After you open it, you can press the letter H to find the relevant instructions about the parameter information, However, it is useless for me, and perhaps more accurately, it is useless for NVDA.
https://docs.microsoft.com/en-us/visualstudio/ide/using-intellisense?view=vs-2019
The following is the feedback I submitted to Microsoft:
https://developercommunity.visualstudio.com/t/Cannot-read-method-parameter-information/1498035?entry=myfeedback&ref=native&refTime=1629131955909&refUserId=e1304cc8-a9db-62bf-a8ac-26749c51abed
I very much hope that NVDA can do this, The ZDSR screen reader from China can support this feature, and it will automatically read out the parameter information when entering a comma. I think this is really good.

### Steps to reproduce:

1. open Visual Studio;
2. create a .net C# project.
3. Type "Console.Beep(,)" in the code editing area (note the comma in parentheses);
4. use Left/Right to move the focus.
5. Stay to the left of the "," (comma);
6. Press Ctrl+K,P (to get the parameter prompt);
7. Stay to the right of the "," (comma)
8. Press Ctrl+K,P (to get the parameter prompt);

### Actual behavior:
Perform steps 6 and steps 8 NVDA will only report "Ready",

### Expected behavior:
NVDA should report help information for the current parameters.
Currently using object navigation can navigate to the parameter prompt in the top-level window of Visual Studio, which is very troublesome. NVDA should automatically report this information when the user types a comma, because they have already appeared on the screen when the comma is typed.

#### NVDA installed/portable/running from source:
installed and portable

#### NVDA version:
NVDA2021.2

#### Windows version:
Windows 10 21H1 (x64) build 19043.1237

#### Name and version of other software in use when reproducing the issue:
Microsoft Visual Studio Community 2019
Version: 16.11.3

#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA2021.1 and the latest Alpha can also reproduce this.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by cary-rowen** [comment]
The following is a discussion in NVDA Groups, which is a common problem:
https://nvda.groups.io/g/nvda/message/87202?p=%2C%2C%2C50%2C0%2C0%2C0%3A%3Arecentpostdate%2Fsticky%2C%2CVISUAL+STUDIO%2C50%2C2%2C0%2C84665360

**Comment by cary-rowen** [comment]
According to Microsoft’s feedback and my test, This issue has been resolved by Microsoft in the preview version. So now i want to close it.
For details, please see the following link:
https://developercommunity.visualstudio.com/t/Cannot-read-method-parameter-information/1498035?entry=myfeedback&ref=native&refTime=1629131955909&refUserId=e1304cc8-a9db-62bf-a8ac-26749c51abed
